Author: Ikey Doherty <ikey@solus-project.com>
Subject: [PATCH] raven: Kill the opacity change on slide-in, looks naff.
Origin: upstream, https://github.com/solus-project/budgie-desktop/commit/c8436288a5af24da31bf0477736f764ef37d26c6
---
 raven/raven.vala | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/raven/raven.vala b/raven/raven.vala
index 989e249..11896ab 100644
--- a/raven/raven.vala
+++ b/raven/raven.vala
@@ -477,11 +477,6 @@ public class Raven : Gtk.Window
                 property = "nscale",
                 old = old_op,
                 @new = new_op
-            },
-            Budgie.PropChange() {
-                property = "opacity",
-                old = old_op,
-                @new = new_op
             }
         };
 
-- 
2.7.4

