From ffe97ae23e22e2603e114d9e36db0b5dca7c2b22 Mon Sep 17 00:00:00 2001
From: fossfreedom <foss.freedom@gmail.com>
Date: Wed, 22 Feb 2017 19:31:05 +0000
Subject: [PATCH] fix crash of the window manager when the current locale is
 deleted

---
 src/wm/keyboard.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/wm/keyboard.vala b/src/wm/keyboard.vala
index 4419665..2128ece 100644
--- a/src/wm/keyboard.vala
+++ b/src/wm/keyboard.vala
@@ -283,7 +283,7 @@ public class KeyboardManager : GLib.Object
     {
         string engine_name;
         InputSource? current = sources.index(current_source);
-        if (current.ibus_engine != null) {
+        if (current != null && current.ibus_engine != null) {
             engine_name = current.ibus_engine;
         } else {
             engine_name = DEFAULT_ENGINE;
-- 
2.7.4

