From b80bb24966094aecd5b5d02f75b3f81f08a054e7 Wed Feb 15 00:00:00 2017
From: David Mohammed <fossfreedom@ubuntu.com>
Date: Wed, 15 Feb 2017 23:17:27 -0000
Subject: [PATCH] patch to support mutter 3.24
Forwarded: not-needed
Origin: Other

Index: budgie-desktop-10.2.9/configure.ac
===================================================================
--- budgie-desktop-10.2.9.orig/configure.ac
+++ budgie-desktop-10.2.9/configure.ac
@@ -65,7 +65,7 @@ PKG_CHECK_MODULES([BUDGIE_PLUGIN],
 # Additional WM deps
 PKG_CHECK_MODULES([BUDGIE_WM],
                 [
-                 libmutter >= mutter_required_version,
+                 libmutter-0 >= mutter_required_version,
                  gnome-desktop-3.0 >= gdesktop_required_version,
                  gsettings-desktop-schemas >= gsettings_required_version,
                  ibus-1.0 >= ibus_required_version
@@ -80,12 +80,12 @@ PKG_CHECK_MODULES([RUNDIALOG],
 )
 
 # Check if we have the mutter private API cruft.
-PKG_CHECK_MODULES(MUTTER_PRIVATE, libmutter >= mutter_goes_private_version,
+PKG_CHECK_MODULES(MUTTER_PRIVATE, libmutter-0 >= mutter_goes_private_version,
         [have_cranky_mutter=yes], [have_cranky_mutter=no]
 )
 
 if test x$have_cranky_mutter = "xyes"; then
-        MUTTER_TYPELIB_DIR=`$PKG_CONFIG --variable=typelibdir libmutter`
+        MUTTER_TYPELIB_DIR=`$PKG_CONFIG --variable=typelibdir libmutter-0`
         AC_SUBST(MUTTER_TYPELIB_DIR)
 fi
 
