Description: Avoid caffeine icon conflicts
Author: Adam Conrad <adconrad@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1786107
Last-Update: 2018-08-08

--- budgie-desktop-10.4+git20180806.01.933f78fc03d.orig/src/applets/caffeine/CaffeineApplet.vala
+++ budgie-desktop-10.4+git20180806.01.933f78fc03d/src/applets/caffeine/CaffeineApplet.vala
@@ -40,7 +40,7 @@ public class Applet : Budgie.Applet
         settings = this.get_applet_settings(uuid);
 
         event_box = new Gtk.EventBox();
-        var icon = new Gtk.Image.from_icon_name("caffeine-cup-empty", Gtk.IconSize.MENU);
+        var icon = new Gtk.Image.from_icon_name("budgie-caffeine-cup-empty", Gtk.IconSize.MENU);
         event_box.add(icon);
         this.add(event_box);
 
--- budgie-desktop-10.4+git20180806.01.933f78fc03d.orig/src/applets/caffeine/CaffeineWindow.vala
+++ budgie-desktop-10.4+git20180806.01.933f78fc03d/src/applets/caffeine/CaffeineWindow.vala
@@ -93,10 +93,10 @@ public class AppletWindow : Gtk.Grid
                 var duration = ngettext ("a minute", "%d minutes", time).printf (time);
                 cmd.append ("\"%s %s\" ".printf (_("Will turn off in"), duration));
             }
-            cmd.append ("--icon=caffeine-cup-full");
+            cmd.append ("--icon=budgie-caffeine-cup-full");
         } else {
             cmd.append ("\"%s\" ".printf (_("Turn off Caffeine Boost")));
-            cmd.append ("--icon=caffeine-cup-empty");
+            cmd.append ("--icon=budgie-caffeine-cup-empty");
         }
 
         try {
@@ -147,7 +147,7 @@ public class AppletWindow : Gtk.Grid
         timer.sensitive = !timer.sensitive;
 
         var state = mode.active ? "full" : "empty";
-        icon = new Gtk.Image.from_icon_name("caffeine-cup-" + state, Gtk.IconSize.MENU);
+        icon = new Gtk.Image.from_icon_name("budgie-caffeine-cup-" + state, Gtk.IconSize.MENU);
         event_box.add (icon);
 
         event_box.show_all();
