Author: Ikey Doherty <ikey@solus-project.com>
Subject: theme: Ensure we link against GTK (issue #465)
Origin: upstream, https://github.com/solus-project/budgie-desktop/commit/6bfadb2ab7d4ef7f8f0bd7859070d29ac64526ad
---
--- a/theme/Makefile.am
+++ b/theme/Makefile.am
@@ -67,6 +67,9 @@ libbudgietheme_la_SOURCES = \
 libbudgietheme_la_CFLAGS = \
 	$(BUDGIE_BASE_CFLAGS)
 
+libbudgietheme_la_LIBADD = \
+	$(BUDGIE_BASE_LIBS)
+
 # Theme headers
 budgie_devdir = $(includedir)/budgie-desktop
 budgie_dev_HEADERS = \
