Description: add build support for vala0.46
Author: David Mohammed <fossfreedom@ubuntu.com>
Origin: https://github.com/solus-project/budgie-desktop/pull/1859
Last-Update: 2019-07-30

--- a/src/applets/places-indicator/ListItem.vala
+++ b/src/applets/places-indicator/ListItem.vala
@@ -20,7 +20,7 @@
     public signal void send_message(string message_content);
     public signal void close_popover();
 
-    public ListItem()
+    protected ListItem()
     {
         orientation = Gtk.Orientation.VERTICAL;
         spacing = 0;
